exercises/practice/resistor-color/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+edition = "2021"
+name = "resistor-color"
+version = "0.0.0"
```

Comment:
In general, an exercise which is live on the site should be version `1.0.0` or higher.